physical/raft/testdata/TestLoadReadOnlySnapshot.bin

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
it looks like this snapshot is different than the enterprise one. Is that intentional?